.meta.toml

```diff
@@ -1,9 +1,9 @@
 # Generated from:
-# https://github.com/plone/meta/tree/main/config/default
+# https://github.com/plone/meta/tree/main/src/plone/meta/default
 # See the inline comments on how to expand/tweak this configuration file
 [meta]
 template = "default"
-commit-id = "a89af8f2"
+commit-id = "2.0.1.dev0"
```

Comment:
Down below there is the `[github]` table, with a `jobs` field which has `test` on it. That should be removed.

Maybe we should add a check on `plone.meta` to warn about it, as that's now handled with the `test_matrix` option.

Reply:
I have removed our custom list.